e2e-tests/commands/shared-commands/assertions/assert-summary-list-row-value.js

```diff
@@ -4,13 +4,21 @@
  * E.g assertSummaryListRowValue({ rowA: {...}, rowB: {...} }, 'startDate', '01/02/2023')
  * @param {Object} Summary list cypress selectors from pages/partials.
  * @param {String} Field ID in the summary list to assert
+ * @param {Number} fieldNumber eg the first or second answer of identical field_id'd rows
```

Comment:
`field_id'd` ?

Reply:
What's the need for this, why would there be multiple identital field IDs rows, shouldn't summary lists have only have unique IDs?